### ISSUE TITLE
Documentation of output files

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/propinquity#6]
Needed for all output directories, but particularly urgent is the `annotated_supertree`. The readme only mentions one file, not three:

     ./annotated_supertree/annotations.json
     ./annotated_supertree/annotations1.json
     ./annotated_supertree/annotations2.json

If there is detailed documentation elsewhere on these (and others) output files, the readme(s) should point to that documentation as well.

**Comment by jar398** [comment]
I have made a pass over the referenced wiki page

https://github.com/OpenTreeOfLife/opentree/wiki/Conflict-and-synthesis-file-formats

to make it clearer that there is an interface here, and I have added an example of conflict/support information, which is incomplete but better than nothing.

Needs more work. In particular the wiki page does not say anything about 'annotations1.json' or 'annotations2.json' and I don't have examples. But maybe these aren't meant to be exposed?
